.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ executors:
     working_directory: ~/repos/geth
   e2e:
     docker:
-      - image: celohq/node10-gcloud:v2
+      - image: celohq/node10-gcloud:v3
```

Comment:
Can you link to the changeset for the docker file?

Reply:
https://github.com/celo-org/celo-monorepo/pull/4282/commits/528dd5e205ea9e3cf598d9b425bf719ea3b04eaa

Reply:
Thanks @bowd 👍